### PR TITLE
[3.1] Credscan followup

### PR DESCRIPTION
**Pull request by aik-jahoda**
Adds suppression messages.

### CODE REVIEW

File: src/System.Data.SqlClient/src/System/Data/SqlClient/TdsEnums.cs [hunk 1]
```diff
@@ -571,6 +571,7 @@ public enum FedAuthLibrary : byte
         // Login data validation Rules
         //
         internal const ushort MAXLEN_HOSTNAME = 128; // the client machine name
+        // [SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="Not a password.")]
```
Comment (danmoseley) [inline src/System.Data.SqlClient/src/System/Data/SqlClient/TdsEnums.cs:574]: should this be on the line below?

**Comment by danmoseley** [comment]
We should not merge any suppressions until we get exemption mail.

Would it make sense to have two PR's, one for suppression and one for PLACEHOLDER changes?

**Comment by aik-jahoda** [comment]
Now it is in two PRs, PLACEHOLDERS needs to be merged prior we can close work items, for suppress messages I think we are fine if we do when we have the exception and  we can merge it in next branch open window. This PR will clean a lot when we merge the #43051.

File: src/System.Data.SqlClient/src/System/Data/SqlClient/TdsEnums.cs (hunk 1 above)
Comment (aik-jahoda) [reply src/System.Data.SqlClient/src/System/Data/SqlClient/TdsEnums.cs:574]: This one is weird. The 
```
        internal const ushort MAXLEN_USERNAME = 128; // the client user id
        internal const ushort MAXLEN_PASSWORD = 128; // ...
```
is considered as password even with the new line (something like `MAXLEN_USERNAME =  MAXLEN_PASSWORD = 128` ) 
This is why the suppression message is on line which doesn't look like password. Perhaps the justification message can be better.

**Comment by danmoseley** [comment]
> Now it is in two PRs,

I'm confused, this PR contains both suppressions and PLACEHOLDERS?

**Comment by aik-jahoda** [comment]
Yep, it is based on last commit of the #43051. When the previous one is merged, this one will contain the suppression only. It is because I needed to run the tool on whole repository with all changes.

**Comment by danmoseley** [comment]
Oh of course - got it!

**Comment by danmoseley** [comment]
Failure is Writing3MBBase64Bytes - unrelated.

**Pull request merged**